src/sqlancer/mysql/gen/MySQLDeleteGenerator.java

```diff
@@ -50,7 +51,13 @@ private SQLQueryAdapter generate() {
                                                     * ignore as a workaround for https://bugs.mysql.com/bug.php?id=95997
                                                     */, "Truncated incorrect INTEGER value",
                 "Truncated incorrect DECIMAL value", "Data truncated for functional index"));
-        // TODO: support ORDER BY
+
+        if (Randomly.getBoolean() && !randomTable.getColumns().isEmpty()) {
+            sb.append(" ORDER BY ");
+            sb.append(randomTable.getRandomNonEmptyColumnSubset().stream()
+                    .map(c -> c.getName() + (Randomly.getBoolean() ? " DESC" : " ASC"))
```

Comment:
The code can be simplified using the following method: https://github.com/sqlancer/sqlancer/blob/7fbfe3d091a6254644dc0f4457e296ada94fe03e/src/sqlancer/mysql/gen/MySQLExpressionGenerator.java#L202